snmp/datadog_checks/snmp/data/default_profiles/hpe-msa.yaml

```diff
@@ -0,0 +1,155 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - _hp-base.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.11.2.51
+metrics:
+  - MIB: FCMGMT-MIB
+    table:
+      OID: 1.3.6.1.3.94.1.8
+      name: connUnitSensorTable
+    symbols:
+      - name: hpe_msa.connUnitSensor
```

Comment:
I'm wondering it it could be better to prefix by a term related to the MIB itself, rather than the profile name.

Do you know what FC means in FCMGMT ? 
(MGMT seems to mean management)

Reply:
Wondering if it could mean `Fibre Channel` as in 
https://www.circitor.fr/Mibs/Html/F/FC-MGMT-MIB.php (but this is owned by Cisco, not hp)

Reply:
Yes! According to [this doc](https://downloads.hpe.com/pub/softlib2/software1/pubsw-linux/p1496432453/v140526/738971-003_SNMP_MIB.html) it stands for Fibre Channel Management MIB
Do you think `fibre_channel` would be a better prefix ? Might not be specific enough

Reply:
1/ What about:
- `hp.fc.connUnitSensor`
- or `hp.fibrechannel.connUnitSensor`
?

I think using fibrechannel (name from MIB) can make it easier (than `msa`) later if we need to move those metrics later to a more generic profile.

2/ Also, about `hpe` vs `hp` in prefix and profile yaml file name prefix, I'm wondering if we should use `hp` only.

Right now, the `hp` profiles yaml names start with both `hp` and `hpe` which seems not great and diverge from the common VENDOR prefix for profile names.

Reply:
1/ `hpe.fibrechannel.connUnitSensor` 
2/ use `hpe-ABC.yaml` for new HPE profiles

Reply:
@AlexandreYang done 🙂